packages/remix-dev/cli/create.ts

```diff
@@ -533,6 +555,24 @@ export async function validateTemplate(input: string) {
 
       switch (response.status) {
         case 200:
+          if (filePath && filePath !== "/") {
+            // if a filePath is included there must also be a branch, because of how github structures
+            // their URLs. That means the api results list all files. Not directories though, so we'll
+            // try to find some file that starts with the path we're after
+            let filesWithinRepo = await response.json();
+            if (
+              !filesWithinRepo.tree.some((file: any) =>
```

Comment:
filesWithinRepo.tree may be undefined (see test failure mentioning `some`) and thus should be tested before access